src/components/Article.jsx

```diff
@@ -1 +1,19 @@
-// export const Article = () => ();
+import React from 'react';
+
+const Article = ({ title, text, date }) => (
+  <div className="box">
+    <h2 className="title" data-cy="title">
+      {title}
+    </h2>
+    <time dateTime="2019-08-30" data-cy="date">
```

Comment:
```suggestion
    <time dateTime={dateISO} data-cy="date">
```
Don't hardcore this value, create a variable for it and calculate it
`const dateISO = date.toISOString().slice(0, 10);`